src/handler/mod.rs

```diff
@@ -564,6 +570,8 @@ impl Default for HandlerConfig {
             viewport: Default::default(),
             context_ids: Vec::new(),
             request_timeout: Duration::from_millis(REQUEST_TIMEOUT),
+            request_intercept: false,
+            cache_enabled: true,
```

Comment:
are these the default values that chrome uses?

Reply:
From Googling I got the impression that it is on by default, it's on my Chrome as well. Judging from CDP protocol it looks like cache is enabled by default since one has to send explicit `SetCacheDisabledParams::new(false || true)` to disable it.